multidim-interop/Makefile

```diff
@@ -1,13 +1,14 @@
 GO_SUBDIRS := $(wildcard go/*/.)
 JS_SUBDIRS := $(wildcard js/*/.)
+RUST_SUBDIRS := $(wildcard rust/*/.)
```

Comment:
This will all go away, now that we are co-locating the test with the implementations, correct?

Reply:
No, we still need to define the steps needed to ask the implementation how to build the container.

I'll mark this PR as draft and it'll make more sense then. Still iterating a bit here.

Reply:
Would it make sense to move this logic into the runner and only reference the commit in `versions.ts`?

The logic is the same (download, unzip, run docker build) for all implementations, right? If we agree on a directory path where in the repo this sits, we could save on the Makefiles for each version.

Reply:
Sounds good, let's see how it pans out! You've dealt with this more than I did :)